### ISSUE TITLE
Add Code version Date Run to Audit file

### ISSUE THREAD

**Issue by TimeScience** [borevitzlab/timestreamlib#171]
This will help us track data consistency and audit our results better. Also will help when publishing because then when we wrap up the results with a DOI, the files will include the required meta data on which version of the code to run to reproduce our results. See next issue for new format suggestions for audit code info at the top of the file

**Comment by Joelgranados** [comment]
This is a great idea. I'm adding information from 172:
Tim Said: 
"""
The idea would be to have a brief json section at the start of the AUDIT file that would list what the code meanings are, the run date and the code version so it could be loaded by the DB and other processes that are tracking code status.
"""

**Comment by Joelgranados** [comment]
I agree that we should explain the errors a bit better, but we should not change the CSV format until we have a compeling reason.
